docs/language/README.md

```diff
@@ -75,11 +80,11 @@ To facilitate both a programming-like model as well as an ad hoc search
 experience, Zed has a canonical, long form that can be abbreviated
 using syntax that supports an agile, interactive query workflow.
 For example, the canonical form of an aggregation uses the `summarize`
-keyword, as in
+reserved word, as in
 ```
 summarize count() by color
 ```
-but this can be abbreviated by dropping the keyword whereby the compiler then
+but this can be abbreviated by dropping `summarize` whereby the compiler then
```

Comment:
@mccanne and I talked about these. The prior use of "keyword" is an accurate parser-centric statement, but it has an unfortunate clash with "keyword search" which is also mentioned prominently in this doc. So here's my attempt to avoid that clash.